resources/Datastore.py

```diff
@@ -19,4 +19,4 @@ def get_type(self):
         if "local" in self.name:
             return "local"
         else:
-            return "Unknown"
+            return "other"
```

Comment:
Didn't you want this as `Unknown`?

Reply:
No. It was `None` before. Then I changed it to `Unknown`, which is not good also. Type = `other` is finally the better choice.